readme.md

```diff
@@ -1,168 +1,161 @@
-- Write ERB views to display AR models
+- Write ERB views to display ActiveRecord models
 - Write forms with attributes for ActiveRecord models
-- Write forms that use a 'namespace' for parameters
+- Write forms that use a namespace for parameters
 - Use `_method` param to emulate PUT and DELETE requests
 
-## Outline
-
-* Why AR + Sinatra
-* Demo Tunr Solution
-* Web app request / response diagram (MVC)
-
-## Why ActiveRecord + Sinatra? (5)
+## Why ActiveRecord and Sinatra?
 
 Sinatra helps us build simple web applications quickly + easily. But we have no
-way to store data. ActiveRecord lets us store data in a DB easily, but has no
-web interface (only CLI).
+way to store data. ActiveRecord lets us store data in a DB easily but has no
+native web interface - only the command line.
 
-Put the two together, and we can build an app with a *web interface* that is
-backed by *data in a database*.
+Put the two together, and we can build an app with a web interface that is
+backed by a database.
 
-Demonstrate Tunr as an example of this.
+> Demonstrate Tunr as an example of this.
 
-## The MVC Paradigm - How Sinatra + AR Work Together (10/15)
+## The MVC Paradigm (15 minutes / 0:15)
 
-1.  The user makes a request (clicking a link, typing in a URL). **Sinatra** receives the request.
-2. **Sinatra** matches the request to the appropriate route (from all the routes we've defined).
-3. That route contains a block of code to run in response (**controller** code). This code is written by us, and determines how to handle that request (unique to our app).
-4. In most cases, the **controller** will use **ActiveRecord** methods to:
-5. create/read/update/destroy data in the **database**.
-6. Once the **controller** has done its job (and gotten any data it needs), it renders a **view** or redirects to a new **request**, which is sent back to the user (their browser) to be displayed.
+How exactly can ActiveRecord and Sinatra work together? Their relationship is best described through the **request-response** cycle...
+
+1.  The user makes a request (e.g., clicking a link, typing in a URL). **Sinatra** receives the request.
+2. **Sinatra** matches the request to the appropriate route that we've defined in our app.
+3. That route contains a block of code to run in response. We call this **controller** code. This code is written by us and determines how to handle that request. This code is unique to our app.
+  - In Sinatra, our routes and controller code live in the same file. This will change with Rails.
+4. In most cases, the controller will use ActiveRecord methods to...
+  - Create, read, update or destroy data in the **database**.
+5. Once the **controller** has done its job and gotten any data it needs, it renders a **view** or redirects to a new **request**, which is displayed back to the user in the browser.
 
 ![Sinatra MVC](images/sinatra_mvc2.jpg)
 
+<!-- AM: Along with diagram, write out quick numbered list indicating order. -->
 
```

Comment:
This is for diagraming the MVC request-response cycle right? Definitely helpful to number it, could turn it into a CFU after to have a student redraw the numbers / explain the steps